doc/ADDB-User-Guide.md

```diff
@@ -0,0 +1,345 @@
+# How to use ADDB
+- Can try it on jupyter notebook on [Chameleon Trovi](https://chameleoncloud.org/experiment/share/84c1d598-699b-4e8d-bee5-6fb7f5937504)
+- Or on a Skylake CENTOS 7.9 baremetal
```

Comment:
Why is this needed? Is ADDB specific or depents on these platforms? Not sure. Please remove, if not.